spec/controllers/application_controller/ci_processing_spec.rb

```diff
@@ -1,4 +1,7 @@
 describe ApplicationController do
+  let!(:server) { EvmSpecHelper.local_miq_server(:zone => zone) }
+  let(:zone) { FactoryGirl.create(:zone) }
```

Comment:
I don't understand.  Were we queueing things without a zone before this change?

Reply:
The zone didn't exist, we would just use `"default"` before in the specs..